src/workers/postUpdated.ts

```diff
@@ -294,19 +301,25 @@ const updatePost = async ({
   }
 
   const title = data?.title || databasePost.title;
-  const updateBecameVisible =
-    content_type === PostType.Freeform
-      ? databasePost.visible
```

Comment:
Not sure why this was here but `should update freeform post and only modify allowed fields` confirms this works still.

Reply:
I think it had to do with freeform are always visible.

Reply:
Yeah I think so too, but now it will be handled in `createPost` since freeform has the title during creation.